File: formulario-backend/formulario/src/main/java/com/bruno/formulario/modulos/Pessoa.java
package com.bruno.formulario.modulos;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "pessoa")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Pessoa {
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    private Long id;
    private String firstName;
    private String lastName;
    private String email;
    private String phone;

}

File: formulario-frontend/atividadeJonatas/src/app/models/pessoa.ts
import { FormGroup } from '@angular/forms';

export class Pessoa {
  id?: any;
  firstName: string;
  lastName: string;
  email: string;
  phone: string;

  constructor(formPessoa: FormGroup) {
    this.id = formPessoa.value.id;
    this.firstName = formPessoa.value.firstName;
    this.lastName = formPessoa.value.lastName;
    this.email = formPessoa.value.email;
    this.phone = formPessoa.value.phone;
  }
}
